server/remote/gcal/event.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package gcal
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+	msgraph "github.com/yaegashi/msgraph.go/v1.0"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
+
+func (c *client) GetEvent(remoteUserID, eventID string) (*remote.Event, error) {
+	if true {
+		return nil, errors.New("gcal GetEvent not implemented")
+	}
+
+	e := &remote.Event{}
+
+	err := c.rbuilder.Users().ID(remoteUserID).Events().ID(eventID).Request().JSONRequest(
+		c.ctx, http.MethodGet, "", nil, &e)
+	if err != nil {
+		return nil, errors.Wrap(err, "msgraph GetEvent")
```

Comment:
s/msgraph/gcal/

Reply:
@levb Notice the beginning block of this function with `if true`, meaning this method is not implemented. The code below the `if true` block is copied directly from the mscalendar implementation, and is only there for reference when the method is to be implemented. This is why it says `msgraph` in this code.

Searching for `if true` in the code reveals 16 spots in my search. So those are basically 16 TODOs on the gcal remote side.